airflow/models/taskinstance.py

```diff
@@ -280,14 +280,13 @@ def clear_task_instances(
                 ti.refresh_from_task(task)
                 if TYPE_CHECKING:
                     assert ti.task
-                task_retries = task.retries
-                ti.max_tries = ti.try_number + task_retries - 1
```

Comment:
we no longer subtract 1 here because we're no longer _adding_ 1 when task is not running.  i assume this old logic would give the wrong result if task was running.....